projects/ccd-case-ui-toolkit/src/lib/shared/components/palette/query-management/components/query-check-your-answers/query-check-your-answers.component.html

```diff
@@ -1,9 +1,15 @@
 <div class="govuk-grid-row">
   <div class="govuk-grid-column-two-thirds-from-desktop">
     <h1 class="govuk-heading-l">
+      <ng-container *ngIf="QueryCreateContext === queryItemTypeEnum.RESPOND; else genericQueryResponseMessage">
+        {{ 'Review query response details' | rpxTranslate }}
+      </ng-container>
+
+      <ng-template #genericQueryResponseMessage>
+        {{ 'Review query details' | rpxTranslate }}
+      </ng-template>
       {{ 'Review query details' | rpxTranslate }}
```

Comment:
Why is {{ 'Review query details' | rpxTranslate }} one within template and one outside